elixir/apps/domain/lib/domain/actors.ex

```diff
@@ -46,6 +46,16 @@ defmodule Domain.Actors do
     end
   end
 
+  def list_groups_for(%Actor{} = actor, %Auth.Subject{} = subject, opts \\ []) do
+    with :ok <- Auth.ensure_has_permissions(subject, Authorizer.manage_actors_permission()) do
+      Group.Query.not_deleted()
+      |> Group.Query.with_joined_memberships()
```

Comment:
It's minor but let's make `Groups.Query.by_actor_id/2` not to leak knowledge about memberships and their bindings here.

Reply:
👍 updated